packages/ncids-css/package.json

```diff
@@ -70,16 +70,13 @@
 			"usa-date-range-picker",
 			"usa-file-input",
 			"usa-fonts",
-			"usa-file-input",
```

Comment:
probably want to remove this change from this PR since this change is in another PR